spec/features/merchants/items/index_spec.rb

```diff
@@ -38,4 +39,39 @@
 
     expect(current_path).to eq(merchant_item_path(@merchant_1, @item_1))
   end
+
+  it "has buttons to enable or disable for each item" do
```

Comment:
Adding # US 8 and a # US 9 can help in terms of test organization and readability when we come back to the project

Reply:
Good catch, this is updated now.